src/components/Screen.vue

```diff
@@ -32,14 +79,18 @@ export default {
       this.currentSlide++;
     },
     onMouseMove(e) {
```

Comment:
Is there a particular need to add a mousemove listener on the `Screen` level, or is it possible to add it only for the specific components that track mouse movement.

Reply:
The purpose of the listener here is that the whole screen area can be tracked.